src/Microsoft.NetCore.Analyzers/Core/Security/DoNotUseInsecureCryptographicAlgorithms.cs

```diff
@@ -102,116 +194,6 @@ private static bool ReferencesAnyTargetType(CompilationSecurityTypes types)
                 || types.RIPEMD160 != null
                 || types.HMACRIPEMD160 != null;
         }
-
-        protected class SyntaxNodeAnalyzer
-        {
-            private readonly CompilationSecurityTypes _cryptTypes;
-
-            public SyntaxNodeAnalyzer(CompilationSecurityTypes cryptTypes)
-            {
-                _cryptTypes = cryptTypes;
-            }
-
-            public void AnalyzeNode(SyntaxNodeAnalysisContext context)
-            {
-                SyntaxNode node = context.Node;
-                SemanticModel model = context.SemanticModel;
-                ISymbol symbol = node.GetDeclaredOrReferencedSymbol(model);
```

Comment:
`GetDeclaredOrReferencedSymbol` - is this API still used by any analyzer? If not, can we delete it? This API scares me.

Reply:
Afraid so.  :-(  Some of the XML analyzers use it, and there's a helper method somewhere under Security\.